plugin.rb

```diff
@@ -229,10 +242,14 @@ def dialogs(options = nil)
         list.more_topics_url = construct_url_with(:next, list_opts)
         list.prev_topics_url = construct_url_with(:prev, list_opts)
 
-        list.draft_key = Draft::NEW_TOPIC
-        list.draft_sequence = DraftSequence.current(current_user, Draft::NEW_TOPIC)
-        list.draft = Draft.get(current_user, list.draft_key, list.draft_sequence) if current_user
-        list
+        if without_respond
```

Comment:
I think that we should set draft attributes only if we need return dialogs items with respond since they needs to show draft dialog form
```suggestion
return list if without_respond

list.draft_key = Draft::NEW_TOPIC
list.draft_sequence = DraftSequence.current(current_user, Draft::NEW_TOPIC)
list.draft = Draft.get(current_user, list.draft_key, list.draft_sequence) if current_user
respond_with_list(list)
```